doc/material-migration.md

```diff
@@ -8,6 +8,33 @@ uid: Uno.Themes.Material.Migration
 
 The Uno Material v5 packages introduce a new dependency on the [Uno Themes](https://www.nuget.org/packages/Uno.Themes.WinUI) package. Uno Themes is the base library for all design system implementations going forward. As a result, the following changes have been made:
 
+### Upgrading to Uno Themes 5.1
+
+#### TextBox `PlaceholderText` and `Header`
```

Comment:
Is it only TextBox, or did we do this for PasswordBox too?

Reply:
Hum, that is a really good point @kazo0!
@eriklimakc I went back to the main issue (https://github.com/unoplatform/Uno.Themes/issues/974) and it is mentioned that changes should normally be done the same for PasswordBox as well. 
But the related PR covers only TextBox: https://github.com/unoplatform/Uno.Themes/pull/1014

Reply:
@agneszitte I think it would be very nice to add more examples for this, as it will give a good comparison between the options. Do you want me to take a look when this is merged?

Reply:
@agneszitte @kazo0 I thought we had agreed to do this only for the TextBox, but I can't find any message or comment confirming that. I might be confusing things. Apologies for missing it. 😞

If agreed by @kazo0, and @rajamatt could help with the PasswordBox that would be great. I can support as/if needed.

Thanks.

Reply:
Issue for the `PasswordBox` work: https://github.com/unoplatform/Uno.Themes/issues/1449